app/src/main/java/org/loklak/android/model/suggest/Query.java

```diff
@@ -0,0 +1,88 @@
+package org.loklak.android.model.suggest;
+
+import com.google.gson.annotations.SerializedName;
+
+
+public class Query {
+
+    private String mQuery;
+    private Integer mQueryCount;
+    private Integer mMessagePeriod;
+    private String mSourceType;
+    private String mRetrievalLast;
+    private Integer mMessagesPerDay;
+    private Integer mQueryLength;
+    @SerializedName("timezoneOffset")
+    private Integer mTimezoneOffset;
+    private String mRetrievalNext;
+    private Integer mScoreRetrieval;
+    private String mQueryLast;
+    private String mExpectedNext;
+    private Integer mScoreSuggest;
+    private Integer mRetrievalCount;
+    private String mQueryFirst;
+
+    public Query() {
```

Comment:
Then can you use the constructor, by putting the initialization of the variables to be done inside the constructor